docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Polkadot 2.0
+description: Explore Polkadot 2.0's innovations making Polkadot's decentralized computer more accessible and efficient.
+---
+
+Understanding what [Polkadot 1.0](./polkadot-v1.md) is about, and the philosophy behind it, will help
+us envision the future direction of the Polkadot as a high-performance decentralized computer capable of delivering secure computation and interoperability to anyone and anything.
+
+## Polkadot as a Decentralized Computer
+
+Polkadot has been abstracted and generalized beyond what was originally proposed and envisioned in
+the [whitepaper](https://polkadot.network/whitepaper/). 
+
+Polkadot 1.0 is:
+
+  - About [**Blockspace**](./polkadot-v1.md#polkadots-blockspace) (the underlying resource that
+  chains provide)
+  - **A platform to build applications** rather than chains, and for people to use those applications.
+  Fundamentally, Polkadot is not a platform to host chains, and so far, chains have happened to be one
```

Comment:
But wasn't the point of parachains to build blockchains, essentially? Or was it always to build applications (and blockchains were just the way to do that)